docs/openapi3.yaml

```diff
@@ -1152,6 +1152,12 @@ paths:
         required: true
         schema:
           type: string
+      - in: query
+        name: dl
+        description: should redownload the file (relevant for file_download typed processors)
+        schema:
+          type: boolean
+          default: false
```

Comment:
add enum: true, false

Reply:
do you mean I should change it to :
```
schema:
    type: string
    enum:
     - true
     - false
```

Reply:
na, leave it as is. The swagger-validator will throw an error if the query param included is not "true" or "false"

Reply:
are you sure? I have tried your suggestion and it didn't act as expected

Reply:
done